commands/htmlCapture.js

```diff
@@ -0,0 +1,48 @@
+
+var fs = require('fs');
+var mkdirp = require('mkdirp');
+
+var mkdirpSync = function(path) {
+    try {
+        mkdirp.sync(path);
+    } catch(e) {
+        if ( e.code != 'EEXIST' ) throw e;
+    }
+}
+
+exports.command = function(fileName, callback) {
+    if (fileName && typeof fileName !== 'string') {
+        throw new Error('htmlCapture expects first parameter to be string; ' + typeof (fileName) + ' given');
+    }
+
+    var htmlCheck = /^\w+\.html$/;
```

Comment:
Saved fixtures will always be .html files. We can probably relax this regex and make it optional for the user to specify ".html" in the file name.